alica_engine/src/engine/AlicaContext.cpp

```diff
@@ -53,19 +53,28 @@ bool AlicaContext::isValid()
     return _validTag == ALICA_CTX_GOOD;
 }
 
-int64_t AlicaContext::getCurrentState() const
+bool isStateActiveHelper(const RunningPlan* rp, int64_t id)
```

Comment:
Can make it a private member of AlicaContext?

Reply:
Made isStateActiveHelper() a private static method of AlicaContext()